task/02-numbers-tasks.js

```diff
@@ -54,7 +54,12 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+    //let max = 1.7976931348623157e+308;
+    let max = Number.MAX_VALUE;
+    if (max < ((value1 + value2) / 2))
+        return 1.7976931348623157e+308;
```

Comment:
Why haven't you used Number.MAX_VALUE here?

Reply:
very sorry i didn't notice